<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VHDesktopSharingUI</class>
 <widget class="QWidget" name="VHDesktopSharingUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>56</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>356</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>356</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widgetTop" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>43</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>43</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#widgetTop
{
background-color: rgba(0, 0, 0, 178);
border 1px solid; 

border-bottom-left-radius:8px;
border-bottom-right-radius:8px;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>43</width>
          <height>43</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>43</width>
          <height>43</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="pushButton_netState">
           <property name="minimumSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-image: url(:/check/images/device_check/icon_net_h.png);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="tBtnLive">
        <property name="minimumSize">
         <size>
          <width>43</width>
          <height>43</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>43</width>
          <height>43</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="tBtnCamera">
        <property name="minimumSize">
         <size>
          <width>43</width>
          <height>43</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>43</width>
          <height>43</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="tBtnMic">
        <property name="minimumSize">
         <size>
          <width>43</width>
          <height>43</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>43</width>
          <height>43</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/button/images/desk_top_mic_5.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="tBtnQuikTool">
        <property name="minimumSize">
         <size>
          <width>43</width>
          <height>43</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>43</width>
          <height>43</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="tBtnSetting">
        <property name="minimumSize">
         <size>
          <width>43</width>
          <height>43</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>43</width>
          <height>43</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_shareSelect">
        <property name="minimumSize">
         <size>
          <width>43</width>
          <height>43</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>43</width>
          <height>43</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">
QPushButton#pushButton_shareSelect{
border-image: url(:/desktopSharing/img/desktopSharing/share_select.png);
}

QPushButton#pushButton_shareSelect:hover{
border-image: url(:/desktopSharing/img/desktopSharing/share_select_hover.png);
}
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="tBtnStopSharing">
        <property name="minimumSize">
         <size>
          <width>43</width>
          <height>43</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>43</width>
          <height>43</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_bottom" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>13</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>13</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QGridLayout" name="layout_bottom">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="1" column="1">
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>2</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>2</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="btnEx">
        <property name="minimumSize">
         <size>
          <width>42</width>
          <height>13</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>42</width>
          <height>13</height>
         </size>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
font-family: &quot;iconfont&quot;;
font:6px;
color: rgb(255, 255, 255);
background-color: rgba(0, 0, 0, 178);
border-bottom-left-radius:3px;
border-bottom-right-radius:3px;

}

QPushButton:hover
{
color: rgb(255, 255, 255);
background-color: #FC5659;
border:1px solid #FC5659;
}

</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>2</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>2</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../VhallLive/project/VhallLive.qrc"/>
  <include location="../Project/MainUI.qrc"/>
 </resources>
 <connections/>
</ui>
